ROMFS/px4fmu_common/init.d-posix/rcS

```diff
@@ -232,15 +232,6 @@ sh etc/init.d/rc.vehicle_setup
 
 # GCS link
 mavlink start -x -u $udp_gcs_port_local -r 4000000
-mavlink stream -r 50 -s POSITION_TARGET_LOCAL_NED -u $udp_gcs_port_local
-mavlink stream -r 50 -s LOCAL_POSITION_NED -u $udp_gcs_port_local
-mavlink stream -r 50 -s GLOBAL_POSITION_INT -u $udp_gcs_port_local
```

Comment:
Does the position and attitude still look smooth in QGC?

Reply:
It should, otherwise we should probably adapt the defaults, right?

Reply:
Exactly. The default for the position is 5 Hz, which is probably too low. I think that was the original reason to add them here.

Reply:
Can we raise it in the mavlink config or would that exceed bandwidth on 57600?

Reply:
We need to check. Why not keep attitude and position here?

Reply:
I think it makes sense to have consistency between SITL and real vehicles as much as possible. And that way we force users like MAVROS, MAVSDK, or QGC to use the commands to request whatever rate they need.